src/client/core/utils/position.js

```diff
@@ -248,28 +221,25 @@ export function getFixedPositionForIFrame (pos, iFrameWin) {
     var iFrameBorders = styleUtils.getBordersWidth(iFrame);
     var iFramePadding = styleUtils.getElementPadding(iFrame);
 
-    return {
+    return offsetToClientCoords({
         x: pos.x - iFrameOffset.left - iFrameBorders.left - iFramePadding.left,
         y: pos.y - iFrameOffset.top - iFrameBorders.top - iFramePadding.top
-    };
+    });
 }
 
-export function getFixedPosition (pos, iFrameWin, convertToClient) {
+export function getIframePointRelativeToParentFrame (pos, iFrameWin) {
     if (!iFrameWin)
```

Comment:
We don't need this check now. And also we can send 'iframe' instead of 'iIframeWin'